### PR TITLE
Add community mananger job description

### PR DESCRIPTION
**Pull request by jackzampolin**
cc @ebuchman

### CODE REVIEW

File: md/careers/community-community-manager.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Cosmos Community Manager
+
+### This position is full time and located in Berlin, Toronto, or the Bay Area. Remote applications will also be considered.
```
Comment (jolesbi) [inline md/careers/community-community-manager.md]: i don't think we are keen on remote applications.

Comment (zramsay) [reply md/careers/community-community-manager.md]: agree

Comment (jackzampolin) [reply md/careers/community-community-manager.md]: IMO for this community manager role specifically we would want to be a bit more flexible for non-remote folks, but thats not up to me!

**Comment by jackzampolin** [comment]
This did not. Can you merge @zramsay

**Comment by zramsay** [comment]
uhm this repo is deprecated ... should go into tendermint.com

**Comment by zramsay** [comment]
https://github.com/tendermint/tendermint.com/pull/180

**Pull request closed without merging**